satpy/readers/gld360_ualf2.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Gld360Ualf2 reader.
+
+Vaisala Global Lightning Dataset GLD360 is data as a service
+that provides real-time lightning data for accurate and early
+detection and tracking of severe weather. The data provided is
+generated by a Vaisala owned and operated world-wide lightning
+detection sensor network.
+
+References:
+- [GLD360] https://www.vaisala.com/en/products/data-subscriptions-and-reports/data-sets/gld360
+
+"""
+
+import logging
+from datetime import timedelta
+
+import dask.dataframe as dd
+import numpy as np
+import xarray as xr
+
+from satpy.readers.file_handlers import BaseFileHandler
+
+logger = logging.getLogger(__name__)
+
+
+class Gld360Ualf2FileHandler(BaseFileHandler):
+    """ASCII reader for Vaisala GDL360 ualf2 data."""
```

Comment:
maybe change to "Reader for the Vaisala Global Lightning Detection Network (GLD360) data in UALF2 format."